domains/fatgrizzly.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "personal website ",
+  "repo": "https://github.com/fatgrizzly",
+  "owner": {
+    "username": "fatgrizzly",
+    "email": "soundar@secret.fyi"
```

Comment:
The domain doesn't exist...

Reply:
The domain doesn't have an A record so it looks like it doesn't exist but it does. 

Domain Lookup:
![image](https://user-images.githubusercontent.com/60828015/114752920-d9006f80-9d24-11eb-9eea-adccbf542bc5.png)

Reply:
sorry...

Reply:
> sorry... really should do more rearch on these domains

Mate, Yes, you need to do a lot, Its an email alias to my main email